src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHost.cs

```diff
@@ -1752,6 +1752,33 @@ private void DoRunspaceInitialization(bool skipProfiles, string initialCommand,
             TelemetryAPI.ReportStartupTelemetry(this);
 #endif
 
+            // If working directory was specified, set it
+            if (s_cpp != null && s_cpp.WorkingDirectory != null)
+            {
+                Pipeline tempPipeline = exec.CreatePipeline();
+                Command c = new Command("Set-Location");
```

Comment:
Personally, I would prefer a better variable name and use var instead.
````c#
var command = new Command("Set-Location");
````

Reply:
Will update